docs/upgrading-to-integration.asciidoc

```diff
@@ -0,0 +1,169 @@
+[[upgrade-to-apm-integration]]
+=== Upgrade to the Elastic APM integration
+
+* <<why-upgrade-to-integrations>>
+* <<apm-arch-upgrade>>
+* <<apm-integration-upgrade-limitations>>
+* <<apm-integration-upgrade-steps>>
+* <<apm-integration-upgrade-steps-ess>>
+
+[discrete]
+[[why-upgrade-to-integrations]]
+=== Why upgrade?
+
+The new APM ecosystem offers a number of benefits:
+
+**{fleet}**:
+
+* A single, unified way to add monitoring for logs, metrics, traces, and other types of data to each host -- install one thing instead of multiple
+* Central, unified configuration management -- no need to edit multiple configuration files
+
+**Data streams**:
+
+* Reduced number of fields per index, better space efficiency, and faster queries
+* More granular data control
+* Errors and metrics data streams are shared with other data sources -- which means better long-term integration with the logs and metrics apps
+* Removes template inheritance for ILM policies and makes use of new {es} index and component templates
+* Fixes <<server-resource-exists-not-alias>>
+
+**APM Integration**:
+
+* Easier to install APM on edge machines
+* Improved source map handling and APM agent configuration management
+* Less configuration
+* Easier and less error-prone upgrade path
+
+[discrete]
+[[apm-arch-upgrade]]
+=== APM integration architecture
+
+The Elastic APM integration consists of four components: *APM agents*, the *Elastic APM integration*, *Elasticsearch*, and *Kibana*.
+Generally, there are two ways that these four components can work together:
+
+Apm agents on edge machines send data to a centrally hosted APM integration:
+
+image::./images/apm-architecture.png[Architecture of Elastic APM]
+
+Or, APM agents and the APM integration live on edge machines and enroll via a centrally hosted {agent}:
+
+image::./images/apm-architecture-two.png[Architecture of Elastic APM option two]
+
+NOTE: If RUM is enabled, you must run {agent} centrally.
+If RUM is disabled, {agent} can be run on edge machines.
```

Comment:
Is this still true?

Reply:
Perhaps a bit pedantic, but you can run it on an edge machine regardless of whether RUM is enabled. Whether you run on the edge or centrally depends on what data you're collecting. Maybe this would be clearer?

> NOTE: in order to collect data from RUM and mobile agents, which run in browser and mobile applications, you must run {agent} centrally. For other applications, such as backend services, {agent} may be co-located on the edge machine.

Reply:
Fixed, thanks